src/properties.rs

```diff
@@ -45,6 +45,11 @@ impl Property {
         self.key.clone()
     }
 
+    /// Clones the value field.
+    pub fn value(&self) -> String {
+        self.value.clone()
+    }
+
```

Comment:
I see there is a precedent by `pub fn key()` returning `String`, but returning `&str` would be much more idiomatic.

`&str` avoids the allocation and copy that is not needed in many cases (and caller can clone whenever needed).

For consistency, this should change signature of `pub fn key()`, which would be an API breaking change per semver. But I think it is worth doing, the library is still in its 0.x stage and every release so far has been API-breaking.

Reply:
I agree – I generally try to make code fit in with the surrounding code (but as Python say "consistency is the hobgoblin of foolish minds" or something like that) :)